doc/api/stream.md

```diff
@@ -1895,6 +1895,87 @@ Calling `Readable.from(string)` or `Readable.from(buffer)` will not have
 the strings or buffers be iterated to match the other streams semantics
 for performance reasons.
 
+### `stream.Readable.fromWeb(readableStream[, options])`
```

Comment:
Is there any reason `from` can't be made to accept a web stream?

Reply:
Discussion here: https://github.com/nodejs/node/pull/39134#issuecomment-876724940